src/main/scala/chisel3/ModuleChoice.scala

```diff
@@ -0,0 +1,95 @@
+// SPDX-License-Identifier: Apache-2.0
+
+package chisel3.choice
```

Comment:
Stylistically, we need to add a directory for this package, so this file should be: `src/main/scala/chisel3/choice/ModuleChoice.scala`